Automattic/wp-calypso
# Add migration sticker and unify the loading experience for the upgrade plan step

Related to #91788

## Proposed Changes

* Add a sticker marking the blog as a migration block when it's loading the upgrade plan step of the migration flow.
* I took the opportunity to unify the loaders of the upgrade plan step, preventing the flickering.
* It doesn't handle request errors while adding the sticker. Should we handle it somehow? Or just proceed with the normal price as workaround? If we want to handle it, I thought of adding some warning or something like this above the plans.

## Why are these changes being made?


* The migration sticker should be added to the blog before proceeding to the checkout step to mark it as a blog that is being migrated.

## Testing Instructions



* Enable the feature flag `migration-flow/introductory-offer`. You can add it to your `.env` file like this `ACTIVE_FEATURE_FLAGS="migration-flow/new-migration-instructions-step,migration-flow/introductory-offer"
`
* Start monitoring your browser requests.
* Start a migration and navigate until the step `site-migration-upgrade-plan`.
* Check that when that step loads, it requests the endpoint `.../{SITE_ID}/migration-flow`

## Recording

https://github.com/Automattic/wp-calypso/assets/876340/96db431a-73f2-4f80-a531-f1b5e73c83a8

## Pre-merge Checklist
